src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/TryHandler.java

```diff
@@ -42,10 +43,31 @@ public TryHandler(IndentationCheck indentCheck,
 
     @Override
     public IndentLevel getSuggestedChildIndent(AbstractExpressionHandler child) {
+
         if (child instanceof CatchHandler
             || child instanceof FinallyHandler) {
             return getIndent();
         }
+
+        final DetailAST resSpecAst = getMainAst().getFirstChild();
+        if (resSpecAst.getType() == TokenTypes.RESOURCE_SPECIFICATION) {
+            final DetailAST resourcesAst = resSpecAst.findFirstToken(TokenTypes.RESOURCES);
+            final IndentLevel expIndent = new IndentLevel(getIndent(), getBasicOffset());
+            DetailAST resourcesAstChild = resourcesAst.findFirstToken(TokenTypes.RESOURCE);
+
+            for (int i = 0; i < resourcesAst.getChildCount(); i++) {
+
+                if (resourcesAstChild.getType() == TokenTypes.RESOURCE
+                    && resourcesAstChild.getLineNo() != resSpecAst.getLineNo()
+                    && resourcesAstChild.getColumnNo()
+                        != Integer.parseInt(expIndent.toString())) {
+                    logError(resourcesAstChild, "",
```

Comment:
line wrapping handler for the resource ast is a good idea. I earlier tried resolving this using checkExprSubtree, but to no avail.